src/pages/tasks/TaskShareDestinationSelectorModal.js

```diff
@@ -5,6 +5,7 @@ import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
 import OptionsSelector from '../../components/OptionsSelector';
+
```

Comment:
Another space added here

Reply:
@thienlnam sorry for the inconvenience, removed some code after the adjustments with the new functions. I thought `npm run prettify` would cover it. Done.